manifests/plugins/java.pp

```diff
@@ -1,4 +1,7 @@
-class collectd::plugins::java ( ) {
+class collectd::plugins::java (
+  $modules,
```

Comment:
remove this as it isn't used and other jmx plugins like cassandra and kafka are failing to install because they don't pass this value (nor do they have a proper module list to pass).